### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for "Azure Signin dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1378]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. Observe the name property of the dialog 

**Actual Result:**
Name property is empty for the Dialog.

**Narrator Behaviour:**
Enter
Authentication Method, selected, Interactive, radio button,

**NVDA:**
dialog  You will manually sign in using your Azure credentials as needed.
Follow these manual steps if Azure Toolkit fails to create an authentication file programmatically.
Azure Sign In
Authentication Method  grouping
Interactive  radio button  checked  1 of 2

**Expected Result:**
Name property should be "Azure Sign in" dialog

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![azure signin-dialog-no name](https://user-images.githubusercontent.com/22956130/35681306-f84a19ee-0782-11e8-8c7c-41ae58672fad.jpg)
![azure signin-dialog-nvda](https://user-images.githubusercontent.com/22956130/35681327-08067bca-0783-11e8-9642-3a973f3cba8c.jpg)

**Comment by Eskibear** [comment]
We just cannot set title bar content for `org.eclipse.jface.dialogs.TitleAreaDialog`. Eclipse doesn't expose the related UI API. `title` of the dialog is already set to "Push Image", which is showing in the title area.

ref: #1407

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
Can show `name ` on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0
![image](https://user-images.githubusercontent.com/6193897/47149981-bf267300-d307-11e8-8242-967bb528fbe9.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749811
